src/main/java/sc/fiji/threed/ThreeDViewer.java

```diff
@@ -46,35 +50,42 @@
 	static Boolean defaultArcBall = true;
 	
 	public ThreeDViewer() {
-		super("ThreeDViewer", 800, 600, false);
+		super("ThreeDViewer", 800, 600, true);
 	}
 	
     public ThreeDViewer(String applicationName, int windowWidth, int windowHeight) {    	
-        super(applicationName, windowWidth, windowHeight, false);        
+        super(applicationName, windowWidth, windowHeight, true);
     }
 
     @Override
     public void init() {
         setRenderer( Renderer.Factory.createRenderer( getHub(), getApplicationName(), getScene(), 512, 512));
-        getHub().add(SceneryElement.RENDERER, getRenderer());
+        getHub().add(SceneryElement.Renderer, getRenderer());
 
         PointLight[] lights = new PointLight[2];
 
         for( int i = 0; i < lights.length; i++ ) {
             lights[i] = new PointLight();
-            lights[i].setPosition( new GLVector(2.0f * i, 2.0f * i, 2.0f * i) );
+            lights[i].setPosition( new GLVector(20.0f * i, 20.0f * i, 20.0f * i) );
             lights[i].setEmissionColor( new GLVector(1.0f, 0.0f, 1.0f) );
-            lights[i].setIntensity( 0.2f*(i+1) );
+            lights[i].setIntensity( 5000.2f*(i+1) );
+            lights[i].setLinear(0.0f);
+            //lights[i].setQuadratic(0.5f);
             getScene().addChild( lights[i] );
         }
 
         Camera cam = new DetachedHeadCamera();
         cam.setPosition( new GLVector(0.0f, 0.0f, 5.0f) );
-        cam.perspectiveCamera(50.0f, getWindowWidth(), getWindowHeight(), 1.0f, 500.0f);
+        cam.perspectiveCamera(50.0f, getWindowWidth(), getWindowHeight(), 0.1f, 10000.0f);
```

Comment:
Take into consideration you are wasting a lot of Z buffer precision here. Maybe use something smaller like 1000 or 500?